extension-base/ctp-extension-gcf-adapter.js

```diff
@@ -3,55 +3,29 @@
 module.exports = {
   ctpExtensionAdapter: function(fn) {
     return function(req, res) {
-      fn(req.body, ctpGcfResponse(res), console.log);
+      fn(req.body, ctpResponse(res), console.log);
     }
   }
 }
 
-function ctpGcfResponse(res) {
+function ctpResponse(res) {
   return {
-    pass: passGcf(res),
-    errors: errorsGcf(res),
-    error: errorGcf(res),
-    updates: updatesGcf(res),
-    update: updateGcf(res),
+    reject: createReject(context),
+    accept: createAccept(context),
   }
 }
 
-function passGcf(res) {
-  return function() {
-    res.status(200).end();
-  }
-}
-
-function errorsGcf(res) {
-  return function(errors) {
-    res.status(400).json({
-      errors : errors
-    });
-  }
-}
-
-function errorGcf(res) {
-  return function(error) {
-    res.status(400).json({
-      errors : [error]
-    });
-  }
+const createReject = context => (error) => {
+  var errorArray = Array.isArray(error) ? error : [error]
+  res.status(400).json({
+    errors : errorArray
```

Comment:
this is super nitpicky, but you could rename `errorArray` to `errors` (we already know that this is a list of errors, thanks to the naming) and use [Object property-value shorthand](https://alligator.io/js/object-property-shorthand-es6/) which came with ES6 out of the box! 😛 

```js
res.status(400).json({
  errors
})

// OR
// inline the ternary
res.status(400).json({
  errors: Array.isArray(error) ? error : [error]
})
```

same for actions/actionsArray